vic/drivers/image/src/vic_force.c

```diff
@@ -192,6 +194,105 @@ vic_force(void)
     // Update the offset counter
     global_param.forceoffset[0] += NF;
 
+    // Initialize the veg_hist structure with the current climatological
+    // vegetation parameters.  This may be overwritten with the historical
+    // forcing time series.
+    for (i = 0; i < local_domain.ncells_active; i++) {
+        for (v = 0; v < options.NVEGTYPES; v++) {
+            vidx = veg_con_map[i].vidx[v];
+            if (vidx != -1) {
+                for (j = 0; j < NF; j++) {
+                    veg_hist[i][vidx].albedo[j] =
+                        veg_con[i][vidx].albedo[dmy[current].month - 1];
+                    veg_hist[i][vidx].LAI[j] =
+                        veg_con[i][vidx].LAI[dmy[current].month - 1];
+                    veg_hist[i][vidx].vegcover[j] =
+                        veg_con[i][vidx].vegcover[dmy[current].month - 1];
+                }
+            }
+        }
+    }
```

Comment:
This overwriting is intentional.  My primary reason was ease of coding (laziness).  Here, in vic_force(), the loop over months of the veg_con climatology can be done easily, and then veg_hist has the climatological values, already "disaggregated" into snow steps, and carries these out to the rest of the code.  If I were to delay this assignment until update_step_vars(), when we are choosing whether to read the values from netcdf files (i.e., no overwrite, just a single write that chooses between veg_con and netcdf), then update_step_vars() would have to know the current month.

So, when we exit vic_force(), the veg_hist arrays are *the* place to look for the LAI etc values, regardless of which option was chosen.

These all result in very simple code in update_step_vars().

This won't affect current performance if you aren't supplying a veg_hist file - this assignment is the same assignment that existed prior to my edits.  If you're reading from a veg_hist file, it does result in a second assignment.  Do you think that that will have a noticeable effect on performance in that case?